src/transforms.js

```diff
@@ -0,0 +1,48 @@
+import _ from 'lodash'
+
+export function transformString (uis1Transform) {
+  const uis2Transform = {
+    from: _.keys(uis1Transform)[0],
+    to: uis1Transform[_.keys(uis1Transform)[0]],
```

Comment:
It looks like you reference `_.keys(uis1Transform)[0],` on the above two lines and thus I'd recommend moving that to a`const`.